src/features/pages/deriv-ctrader/other-platforms/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react'
+import dxIcon from 'images/svg/deriv-ez/dx-icon.svg'
+import dtIcon from 'images/svg/deriv-ez/dt-icon.svg'
+import dmt5Icon from 'images/svg/deriv-ez/dmt5-icon.svg'
+import dGoIcon from 'images/svg/deriv-ez/go-icon.svg'
+import dbIcon from 'images/svg/deriv-ez/db-icon.svg'
+import GenericCarousel, { TRenderableData } from 'components/custom/carousel/_platforms-carousel'
+
+const OtherPlatforms = () => {
+    const [is_loaded, setLoaded] = useState(false)
+
+    useEffect(() => {
+        // added force rerendering because whole page is on new approach and only this component is on old approach. Need to refactor
+        setLoaded(true)
+    }, [])
+
+    const carouselData: TRenderableData[] = [
```

Comment:
this array has no dependencies, there is not state or props being passed to it, can you please move it to outside of the component? 
thanks :pray:

Reply:
Hi @mohsen-deriv , Abu hasan is working on this component in new approach. Once it's ready we will replace this component.